src/components/pagination/pagination.classnames.ts

```diff
@@ -0,0 +1,72 @@
+import { cva } from 'class-variance-authority'
+
+const paginationBaseClass =
+  'focus-visible:outline-offset-2 focus-visible:outline-solid-blue-700 disabled:pointer-events-none flex items-center justify-center'
+
+const buttonTextBaseClass = 'text-ellipsis leading-100'
+
+export const paginationVariants = cva(paginationBaseClass, {
+  variants: {
+    variant: {
+      number:
+        'h-1600 w-1600 rounded-200 hover:bg-opacity-black-100 active:bg-opacity-black-200',
+      bullet: 'rounded-pill',
+    },
+    theme: {},
```

Comment:
do we need this variant?

Reply:
Currently light and dark variant is not provided in Figma. Robin is yet to develop them.